suite-native/navigation/src/navigators.ts

```diff
@@ -24,9 +24,12 @@ type AddCoinFlowParams = RequireAllOrNone<
     'networkSymbol' | 'accountType' | 'accountIndex'
 >;
 
-type ReceiveAccountsParams = {
+export type CloseActionType = 'back' | 'close';
```

Comment:
Do we need this type at all? Because everywhere it is used, you always pass `close`. What about omitting this route parameter and hardcode `close` to those components that now read it from there? 

Or is it a preparation for another issue?

Reply:
This is to differentiate the close button based on from which flow the route was called. The same route should render `X` if you get there from Add Coin flow and `<` if you get there from any other flow.

Reply:
Ok, but you are not passing `back` anywhere right? So you will add it in another PR?

Reply:
Correct, I am only passing 'close'

Reply:
Not planning to pass the other one, but passing something like `showCloseAction` felt inferior to this solution